src/vm/i386/excepx86.cpp

```diff
@@ -3708,3 +3708,34 @@ AdjustContextForVirtualStub(
 
     return TRUE;
 }
+
+//Return TRUE if pContext->Pc is in VirtualStub
+BOOL IsIPinVirtualStub(PCODE f_IP)
```

Comment:
Instead of having identical copy of this function at two places (one for ARM and one for x86), could you please move it to the exceptionhandling.cpp instead so that we have just one copy?